src/containers/ListingPage/ListingPage.js

```diff
@@ -497,7 +497,7 @@ ListingPageComponent.defaultProps = {
   showListingError: null,
   reviews: [],
   fetchReviewsError: null,
-  timeSlots: [],
+  timeSlots: null,
```

Comment:
The default `timeSlots` value on listing page level is changed to `null` so that it can be used to resolve if time slots are used. An empty array that would be passed to subcomponents would just render a blocked calendar.